.github/workflows/build.yaml

```diff
@@ -0,0 +1,288 @@
+             
+        - name: Display Galasa Javadoc pom.xml 
+          run: |
+            cat ${{github.workspace}}/obr/javadocs/pom.xml
+          
+        - name: Build javadoc site using maven
+          run: |
+            mvn -f ${{github.workspace}}/obr/javadocs/pom.xml deploy \
+            -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/managers \
+            -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+            -Dgalasa.release.repo=file:${{ github.workspace }}/obr/javadocs/docker/repo \
+            -Dmaven.javadoc.failOnError=false --batch-mode --errors --fail-at-end \
+            --settings  /home/runner/work/gpg/settings.xml
+        
+        - name: Login to Github Container Registry
+          uses: docker/login-action@v3
+          with:
+            registry: ${{ env.REGISTRY }}
+            username: ${{ github.actor }}
+            password: ${{ secrets.GITHUB_TOKEN }}
+      
+        - name: Extract metadata for Javadocsite image
+          id: metadata-javadocsite
+          uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+          with:
+            images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/javadocsite
+    
+        - name: Build and Push Javadocsite image 
+          id: build-javadocsite
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ${{github.workspace}}/obr/dockerfiles/dockerfile.javadocsite
+            push: true
+            tags: ${{ steps.metadata-javadocsite.outputs.tags }}
+            labels: ${{ steps.metadata-javadocsite.outputs.labels }}
+        
+        - name: Recycle javadocsite application in ArgoCD
+          run: |
+            docker run --env ARGOCD_AUTH_TOKEN=${{ env.ARGOCD_AUTH_TOKEN }} --rm -v ${{ github.workspace }}:/var/workspace ghcr.io/galasa-dev/argocdcli:main app actions run gh-maven-repos restart --kind Deployment --resource-name javadocsite-gh --server argocd.galasa.dev
+        
+        - name: Extract metadata for JavadocMavenrepo image
+          id: metadata
+          uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+          with:
+            images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/JavadocMavenrepo
+      
+        - name: Build and Push JavadocMavenrepo image 
+          id: build
+          uses: docker/build-push-action@v5
+          with:
+            context: .
+            file: ${{github.workspace}}/obr/dockerfiles/dockerfile.javadocmavenrepo
+            push: true
+            tags: ${{ steps.metadata.outputs.tags }}
+            labels: ${{ steps.metadata.outputs.labels }}
+            build-args: |
+              dockerRepository=${{env.REGISTRY}}
+              tag=latest

```

Comment:
This should be `baseVersion=latest` - please also make this same change in the PR workflow

Reply:
done